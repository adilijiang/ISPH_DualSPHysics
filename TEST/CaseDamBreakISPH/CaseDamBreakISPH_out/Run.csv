#RunName;RunCode;DateTime;Np;TSimul;TSeg;TTotal;MemCpu;MemGpu;Steps;PartFiles;PartsOut;MaxParticles;MaxCells;Hw;StepAlgo;Kernel;Viscosity;ViscoValue;DeltaSPH;TMax;Nbound;Nfixed;H;RhopOut;PartsRhopOut;PartsVelOut;CellMode;RunMode;VA-Init;NL-Limits;NL-MakeSort;NL-SortData;NL-OutCheck;CF-PreForces;CF-Forces;SU-Shifting;SU-ComputeStep;SU-Floating;SU-Motion;SU-Periodic;SU-ResizeNp;SU-SavePart
CaseDamBreakISPH;vh911anc;04-11-2015 13:07:49;21001;0.194637;194.637131;0.219006;4578620;0;10;10;0;21001;2664;Cpu;Symplectic;Wendland;Laminar+SPS;0.020000;0;0.001000;1001;1001;0.014142;(700-1300);0;0;2H;Pos-Simple, OpenMP(Threads:8);0.024487;0.003213;0.004591;0.008513;0.000001;0.017135;0.084105;0.000000;0.017163;0.000000;0.000000;0.000000;0.000000;0.059358
