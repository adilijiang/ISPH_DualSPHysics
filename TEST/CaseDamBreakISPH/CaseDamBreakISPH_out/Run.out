

<DUALSPHYSICS> Copyright (C) 2015 by Dr Jose M. Dominguez, Dr Alejandro Crespo,
Prof. M. Gomez Gesteira, Dr Anxo Barreiro, Dr Ricardo Canelas, Dr C. Altomare
Dr Benedict Rogers, Dr Georgios Fourtakas, Dr Athanasios Mokos,
Dr Stephen Longshaw, Dr Renato Vacondio

EPHYSLAB Environmental Physics Laboratory, Universidade de Vigo
School of Mechanical, Aerospace and Civil Engineering, University of Manchester

DualSPHysics is free software: you can redistribute it and/or
modify it under the terms of the GNU General Public License as
published by the Free Software Foundation, either version 3 of
the License, or (at your option) any later version.

DualSPHysics is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License,
along with DualSPHysics. If not, see <http://www.gnu.org/licenses/>.


DualSPHysics Re v4.0.015 (03-10-2015)
======================================
Threads by host for parallel execution: 8
[Initialising JSphCpuSingle v0.70  04-11-2015 13:07:49]

*** Attention: Eps value is not used (this correction is deprecated).

**Basic case configuration is loaded
**Special case configuration is loaded
Loading initial state of particles...
Loaded particles: 21001
MapRealPos(border)=(-1.41421e-005,-1.41421e-005,-1.41421e-005)-(4.00001,1.41421e-005,3.00001)
MapRealPos(final)=(-1.41421e-005,-1.41421e-005,-1.41421e-005)-(4.00001,1.41421e-005,4.50003)
**Initial state of particles is loaded
**2D-Simulation parameters:
CaseName="CaseDamBreakISPH"
RunName="CaseDamBreakISPH"
SvDouble=False
SvTimers=True
SvTimersStep=0.000000
StepAlgorithm="Symplectic"
Kernel="Wendland"
Viscosity="Laminar+SPS"
Visco=0.020000
ViscoBoundFactor=1.000000
DeltaSph="None"
Shifting="None"
RenCorrection=0.000000
FloatingFormulation="None"
FloatingCount=0
CaseNp=21001
CaseNbound=1001
CaseNfixed=1001
CaseNmoving=0
CaseNfloat=0
CaseNfluid=20000
PeriodicActive=0
Dx=0.01
H=0.014142
CoefficientH=1
CteB=1115537.125000
Gamma=7.000000
RhopZero=1000.000000
Eps=0
Cs0=88.3672
CFLnumber=0.200000
DtIni=0.0001
DtMin=1e-005
DtAllParticles=False
MassFluid=0.100000
MassBound=0.100000
Bwen (wendland)=-984716.812500
SpsSmag=0.000001
SpsBlin=0.000000
TimeMax=0.001
TimePart=0.0001
Gravity=(0.000000,0.000000,-9.810000)
NpMinimum=1001
RhopOut=True
RhopOutMin=700.000000
RhopOutMax=1300.000000
**Requested cpu memory for 21001 particles: 3.5 MB.
CellOrder="XYZ"
CellMode="2H"
Hdiv=1
MapCells=(142,1,160)
DomCells=(142,1,160)
DomCellCode="13_7_12"
**CellDiv: Requested cpu memory for 22051 particles: 0.7 MB.
**CellDiv: Requested cpu memory for 5548 cells (CellMode=2H): 0.1 MB.
RunMode="Pos-Simple, OpenMP(Threads:8)"
Allocated memory in CPU: 4578620 (4.37 MB)
Part_0000        21001 particles successfully stored

[Initialising simulation (vh911anc)  04-11-2015 13:07:49]
PART       PartTime      TotalSteps    Steps    Time/Seg   Finish time        
=========  ============  ============  =======  =========  ===================
Part_0001      0.000100             1        1     213.36  04-11-2015 13:07:49
Part_0002      0.000200             2        1     200.07  04-11-2015 13:07:49
Part_0003      0.000300             3        1     202.78  04-11-2015 13:07:49
Part_0004      0.000500             5        2     167.06  04-11-2015 13:07:49
Part_0005      0.000600             6        1     202.07  04-11-2015 13:07:49
Part_0006      0.000700             7        1     196.13  04-11-2015 13:07:49
Part_0007      0.000800             8        1     198.02  04-11-2015 13:07:49
Part_0008      0.000900             9        1     199.36  04-11-2015 13:07:49
Part_0009      0.001000            10        1     187.62  04-11-2015 13:07:49

[Simulation finished  04-11-2015 13:07:49]
Particles of simulation (initial): 21001
DTs adjusted to DtMin............: 0
Excluded particles...............: 0
Total Runtime....................: 0.219006 sec.
Simulation Runtime...............: 0.194637 sec.
Time per second of simulation....: 194.637131 sec.
Steps per second.................: 51.377659
Steps of simulation..............: 10
PART files.......................: 10
Maximum number of particles......: 21001
Maximum number of cells..........: 2664
CPU Memory.......................: 4578620 (4.37 MB)

[CPU Timers]
VA-Init..........................: 0.024487 sec.
NL-Limits........................: 0.003213 sec.
NL-MakeSort......................: 0.004591 sec.
NL-SortData......................: 0.008513 sec.
NL-OutCheck......................: 0.000001 sec.
CF-PreForces.....................: 0.017135 sec.
CF-Forces........................: 0.084105 sec.
SU-Shifting......................: 0.000000 sec.
SU-ComputeStep...................: 0.017163 sec.
SU-Floating......................: 0.000000 sec.
SU-Motion........................: 0.000000 sec.
SU-Periodic......................: 0.000000 sec.
SU-ResizeNp......................: 0.000000 sec.
SU-SavePart......................: 0.059358 sec.
 
